src/docs/swagger.ts

```diff
@@ -34,7 +34,7 @@ const options: swaggerJsdoc.Options = {
       },
     },
   },
-  apis: ['./src/docs/documentation.ts', './src/docs/auth.docs.ts', './src/docs/user.docs.ts'],
+   apis: ['./src/docs/documentation.ts', './src/docs/*.docs.ts'],
```

Comment:
I think it will be better if we all follow the same structure. reference  on @martinemahirwe 's PR